plugins/source/azure/Makefile

```
@@ -24,6 +24,10 @@ gen-docs:
 	rm -rf ./docs/tables/*
 	go run main.go doc ./docs/tables
 
+.PHONY: lint
+lint:
+	golangci-lint run
```

```suggestion
	golangci-lint run --config ../../.golangci.yml 
```

We need to point to the shared config 2 levels up to match what we do in CI